rafthttp/transport.go

```diff
@@ -33,6 +34,10 @@ type Raft interface {
 	ReportSnapshot(id uint64, status raft.SnapshotStatus)
 }
 
+type ClusterVersion interface {
```

Comment:
It is the type of one argument in NewTransporter function, so it becomes public.